Orange/widgets/unsupervised/owlouvainclustering.py

```diff
@@ -190,6 +208,7 @@ def cancel(self):
         self.__set_state_ready()
 
     def commit(self):
+        # pylint: disable=too-many-branches
```

Comment:
I see no reason why this pylint check _should_ be disabled for this function.
And I kind of agree with pylint that this function has a lot of `if`s... Can we reduce them?
E.g. looking at the `len(...attributes) < 1` check - wouldn't a better place for this be in set_data (and on error just set self.data to None, so it does not have to be checked ever again).